pkg/cmd/pipelinerun/describe_test.go

```diff
@@ -2069,3 +2069,147 @@ func TestPipelineRunDescribeV1beta1_withoutNameOfOnlyOnePipelineRunPresent(t *te
 	}
 	golden.Assert(t, got, fmt.Sprintf("%s.golden", t.Name()))
 }
+
+func TestPipelineRunDescribe_status_succeeded(t *testing.T) {
+	clock := clockwork.NewFakeClock()
+
+	trs := []*v1alpha1.TaskRun{
+		tb.TaskRun("tr-1",
+			tb.TaskRunNamespace("ns"),
+			tb.TaskRunStatus(
```

Comment:
instead of `TestBuilders` can we use `structs`? Probably you can refer [here](https://github.com/tektoncd/cli/pull/1289/files#diff-5d2d42d5d1d382c94ce1506f9f75a226d74874d8b62d9af318b93b063109e158R66-R72) for sample conversion from test builder to struct.

Reply:
Fixed.